client/pages/login.vue

```diff
@@ -45,35 +50,53 @@
 
 <script>
 import Vue from 'vue'
-import debounce from '@/utils/debounce'
 import Component from 'class-component'
+import debounce from '@/utils/debounce'
 
-@Component
+@Component({
+  head () {
+    return {
+      title: 'Sign In'
+    }
+  }
+})
 export default class Login extends Vue {
   user = {
     userName: '',
     password: '',
-    captcha: ''
+    captcha: '',
+    showAccessToken: true
   }
-  authenticated = false // #WatchHowDoWe ... How do we?
   rules = {}
   captchaSvg = ''
   // keepPwd = false
   logging = false
+
   layout () {
     return 'empty'
   }
   mounted () {
     this.getCaptcha()
   }
+  async asyncData ({
+    redirect,
+    query,
+    store
+  }) {
+    let path = query.page || '/'
+    let authenticated = await store.getters['session/authenticated']
+    if (authenticated) {
+      redirect(path)
```

Comment:
So we can redirect to `/` or elsewhere (e.g. bookmarked link as `/login?page=%2Fabout` => `/about` if already signed in and accessed `/login`.